### PR TITLE
UI: Configuration, documentation, & minor fixes

### LINKED ISSUE #1104: UI: allow hostnames/dns entries
## Specify type:
- Bug

### Priority:
- Medium

___

## Description:
In the Add Source wizard, the Address fields should allow hostnames/dns entries. This is true for Network hosts, Satellite, and vCenter.
___

### PR DESCRIPTION
**Pull request by cdcabrera**
Config
* Configuration for AJAX timeouts

Docs & Fixes
* Additional Docker ReadMe updates for Linux
* Error response string condition fix
* Remove unordered listing for single network host
* Allow hostname checks, fixes #1104

Prep for testing updates
* ~Added an index file for actions~
* Removed unused navigationBar constants
* Sorted ~actions~, reducers

Note: ~Temporarily leaving this as multiple commits,~ Reserving the styling update for another iteration. The primary fix is implemented for #1104 , minor styling update remains.

updates #1017 closes #1104

### CODE REVIEW

File: client/src/components/sources/sourceListItem.js [hunk 1]
```diff
@@ -255,12 +255,16 @@ class SourceListItem extends React.Component {
 
     const itemHostsPopover = (
       <Popover id={helpers.generateId()} className="quipucords-sources-popover-scroll">
```
[the remark this answers is not in the document]
Comment (cdcabrera) [reply client/src/components/sources/sourceListItem.js]: Can do!

File: client/src/redux/reducers/index.js [hunk 2]
```diff
@@ -1,33 +1,47 @@
...
+  credentialsReducer,
+  factsReducer,
+  reportsReducer,
+  scansReducer,
+  sourcesReducer,
+  statusReducer,
+  toastNotificationsReducer,
+  userReducer,
+  viewOptionsReducer
+};
```
Comment (jeff-phillips-18) [inline client/src/redux/reducers/index.js]: Is there a reason to export individual reducers?

File: client/src/redux/actions/index.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  actions,
+  credentialsActions,
+  factsActions,
+  reportsActions,
+  scansActions,
+  sourcesActions,
+  statusActions,
+  userActions
+};
+
```
Comment (jeff-phillips-18) [inline client/src/redux/actions/index.js]: Is there a plan to import actions differently?

File: client/src/redux/reducers/index.js (hunk 2 above)
Comment (cdcabrera) [reply client/src/redux/reducers/index.js]: Looks like the only time it pops up is rolling through the unit testing. Did notice we cleaned it up once, I'll revert it back down to just exporting the `reduxReducers, reducers`... and follow it up with a similar refinement for the actions too

File: client/src/redux/actions/index.js (hunk 3 above)
Comment (cdcabrera) [reply client/src/redux/actions/index.js]: So was looking to make it consistent, namespacing for everything; reducers, actions, constants. From there it's the convenience of simply going that's an `action` prefix and possible intellisense on your IDE/editor

Comment (cdcabrera) [reply client/src/redux/actions/index.js]: Per our side discussion, going to remove this `index.js` file and reserve it for a testing pr if it becomes necessary

**Pull request merged**